.pre-commit-config.yaml

```diff
@@ -13,10 +13,10 @@ repos:
       - id: check-merge-conflict
   # NOTE: Keep order as pycln (remove unused imports), then isort (sort them) and black (final formatting)
   - repo: https://github.com/hadialqattan/pycln
-    rev: v1.1.0
+    rev: v1.2.4
     hooks:
       - id: pycln
-        args: [--all]
+        args: [--all, --expand-stars]
```

Comment:
ok I guess I had to tackle the same issue that you did. There was a ```from XXX import *``` in one test right? I fixed it by removing it since it was actually unnecessary. Do we want to allow star imports in tests or not? I would favor the latter to reduce unwanted imports that may screw tests... but I'm open to discussion

Reply:
What ``--expand-stars`` do is to move from ``from something import *`` to ``from something import a, b`` (pycln automatically finds what to import) ... 

>  Do we want to allow star imports in tests or not? 

I vote for no. The main reason is that in hides collisions between function and variable names that are difficult to catch. Pylint also disallows it. But now you can do it while developing and let pycln to expand it for you

Reply:
I see. ok, anyway I am also against ```*```